### PR TITLE
Reduced shopping list to the items that were missing

### PR DESCRIPTION
**Pull request by megaptera-helvetiae**
Following from #16, @megaptera-helvetiae forgot to shop some items yesterday. She needs to go back to Rewe. I deleted all the items that we have and only left graded cheese and dog food on the [list](https://github.com/pbuttigieg/shoppingList/blob/main/lists/LaetisShoppingList.Md).

### CODE REVIEW

File: lists/LaetisShoppingList.Md [hunk 1]
```diff
@@ -1,9 +1,3 @@
-milk
-raisin bread
-fresh spinach leaves
-cheese
 dog food
```
Comment (pbuttigieg) [inline lists/LaetisShoppingList.Md:1]: Missing a "-"

File: lists/LaetisShoppingList.Md [hunk 2]
```diff
@@ -1,9 +1,3 @@
...
-wet wipes
-potatoes
 graded cheese
```
Comment (pbuttigieg) [inline lists/LaetisShoppingList.Md:2]: * Missing a "-"
* Typo: "grated"

**Review by pbuttigieg — CHANGES_REQUESTED** [review]
Hi @megaptera-helvetiae some minor revisions needed.

File: lists/LaetisShoppingList.Md (hunk 2 above)
Comment (megaptera-helvetiae) [reply lists/LaetisShoppingList.Md]: @pbuttigieg please see my [edits](1bdb9535e20e58ae3063af85fadc98ee8f4501cf). I grated the cheese and added the lines.

**Comment by pbuttigieg** [comment]
@megaptera-helvetiae looks good to merge

**Pull request merged**